prow/plugins/welcome/welcome.go

```diff
@@ -32,7 +32,7 @@ import (
 
 const (
 	pluginName            = "welcome"
-	defaultWelcomeMessage = "Welcome @{{.AuthorLogin}}! It looks like this is your first PR to {{.Org}}/{{.Repo}} 🎉"
+	defaultWelcomeMessage = "Welcome @{{.AuthorLogin}}! It looks like this is your first PR to {{.Org}}/{{.Repo}} 🎉. Please refer to our [pull request process documentation](https://git.k8s.io/community/contributors/guide/pull-requests.md) to help your PR have a smooth ride to approval. You will be prompted by a bot to use commands during the review process. Do not be afraid to follow the prompts! It is okay to experiment. [Here is the bot commands documentation](https://go.k8s.io/bot-commands). You can also check if {{.Org}}/{{.Repo}} has [its own contribution guidelines]({{.Org}}/{{.Repo}}/CONTRIBUTING.md). You may want to refer to our [testing guide](https://git.k8s.io/community/contributors/devel/sig-testing/testing.md)if you run into trouble with your tests not passing. If you are having difficulty getting your pull request seen, please follow the [recommended escalation practices](https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#why-is-my-pull-request-not-getting-reviewed). We want to make sure your contribution gets all the attention it needs! Thank you, and welcome to Kubernetes."
```

Comment:
rendered:
-----
Welcome @guineveresaenger ! It looks like this is your first PR to kubernetes/test-infra 🎉. Please refer to our [pull request process documentation](https://git.k8s.io/community/contributors/guide/pull-requests.md) to help your PR have a smooth ride to approval. You will be prompted by a bot to use commands during the review process. Do not be afraid to follow the prompts! It is okay to experiment. [Here is the bot commands documentation](https://go.k8s.io/bot-commands). You can also check if kubernetes/test-infra has [its own contribution guidelines]kubernetes/test-infra/CONTRIBUTING.md). You may want to refer to our [testing guide](https://git.k8s.io/community/contributors/devel/sig-testing/testing.md)if you run into trouble with your tests not passing. If you are having difficulty getting your pull request seen, please follow the [recommended escalation practices](https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#why-is-my-pull-request-not-getting-reviewed). We want to make sure your contribution gets all the attention it needs! Thank you, and welcome to Kubernetes.

Reply:
- this might be a bit overwhelming?
  - maybe add line breaks? EG we could split out the actual welcome portion from all of the helpful info by a blank line :thinking: 
- maybe linkify all of the `{{.Org}}/{{.Repo}}` references?
  - the repo specific contribution guidelines link looks broken
- there is at least one missing spaces I think

Reply:
How did you render this? I was puzzled by how to do that 😅

Reply:
I'm missing some kind of way to escape that open paren after the repo specific link? :/

Reply:
- copy the string contents to my editor
- replace template variable references with real values
- paste into github
A better way to do this would be to actual run it through the template engine but the template is simple enough that I didn't because :effort: 😅